### ISSUE TITLE
Add reservedAttributes to adapter.js template

### ISSUE THREAD

**Issue by dmarcelino** [balderdashy/sails-generate-adapter#8]
`reservedAttributes ` allows an adapter to define a whitelist of attributes that are special to it, these attributes won't be stripped by waterline. Full details on:
Details on https://github.com/balderdashy/waterline/issues/303#issuecomment-37418492

**Comment by mikermcneil** [comment]
Thanks for posting, @dmarcelino.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails-generate-adapter/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
